data/dbus/org.fedoraproject.Anaconda.Boss.conf

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE busconfig PUBLIC
```

Comment:
That should be doable, but at the same time I wonder if it is worth it at the moment vs just have them as static files we change as needed, at least initially.

Also even if we generate them I think it might make sense to still have them checked in in the source code repository, so it's easy to reference them if some issues arise.

Otherwise people would have to fish for them in binary packages or on installation images.